SSO-Auth/Api/SSOController.cs

```diff
@@ -384,6 +393,15 @@ public async Task<ActionResult> SamlAuth([FromBody] AuthResponse response)
                 }
                 var authenticationResult = await Authenticate(samlResponse.GetNameID(), isAdmin, config.EnableAuthorization, config.EnableAllFolders, folders.ToArray(), response)
                     .ConfigureAwait(false);
+                    
+                if(config.DefaultProvider != "")
```

Comment:
This should not be here, and instead should be moved into the `Authenticate` function.